stable/fairwinds-insights/values.yaml

```diff
@@ -39,6 +42,8 @@ options:
   autogenerateKeys: false
   # -- Run the job to migrate health scores to a new format
   migrateHealthScore: true
+  # -- Name of the secret where session keys and other secrets are stored
+  secretName: fwinsights-secrets
```

Comment:
ditto ... on docs

Reply:
By the way you need to add `.values.SecretName` in secret https://github.com/FairwindsOps/charts/blob/master/stable/fairwinds-insights/templates/secret.yml#L10 it is still hardocoded one ... so even if new value is supplied here it will fail still.

Reply:
Good catch!